lib/semantics/check-omp-structure.cc

```diff
@@ -585,9 +585,6 @@ void OmpStructureChecker::Enter(
     OmpClauseSet allowed{OmpClause::DEPEND};
     SetContextAllowed(allowed);
   } break;
-  default:
-    // TODO others
-    break;
```

Comment:
This switch statement should also have `SWITCH_COVERS_ALL_CASES` as per the updated style guide

Reply:
@klausler looks like @DavidTruby's comment fell through the cracks. @DavidTruby - can you submit a separate PR for it?